zerver/openapi/zulip.yaml

```diff
@@ -3869,15 +3875,6 @@ paths:
                                       description: |
                                         The [policy](/api/roles-and-permissions#permission-levels)
                                         for which users can create bot users in this organization.
-                                    community_topic_editing_limit_seconds:
-                                      type: integer
-                                      description: |
-                                        Messages sent more than this many seconds ago cannot have
-                                        their topics edited by other users with this organization's
-                                        [message edit policy](/help/restrict-message-editing-and-deletion).
-
-                                        **Changes**: New in Zulip 3.0 (feature level 11). Previously this
-                                        value was hardcoded to 86400 seconds (1 day).
```

Comment:
This should not have been present here in the events documentation for `realm/update_dict` events, since it is a constant and we do not include this field in the events for changing other settings as well. Not sure how we want to handle the changes entry for this.

Reply:
If it's never been returned by this event, then we could probably drop it completely without a **Changes** note?

Or maybe (above):
> **Changes**: In Zulip 7.0 (feature level 183), the realm setting `community_topic_editing_limit_seconds` was removed. It was documented as potentially returned as a changed property in this event, but in fact it was only ever returned in the [`POST /register`](/api/register-queue) response.

@gnprice - It might be good to have your thoughts here as we're removing documentation that was inaccurate. Does that warrant a **Changes** note? If not, then perhaps a note in the commit message about why we're not documenting this as an API change?

Reply:
Since we had this property documented for a while, it'd be good to have a note somewhere on the page about why it no longer appears — even if in reality it never appeared in the server's actual responses, and only existed in the docs.

That's because existing client code may refer to it; which means when someone goes to update that code and compares it to the current docs, they'll notice the discrepancy and may want to track down what happened. It's best if Ctrl+F on the page is enough to find an answer to that question.

(For example, zulip-mobile has a realm setting `community_topic_editing_limit_seconds` in its type definitions for the API.)

A "Changes" note at the parent object of where the property would have appeared would be a good place for that information.

Reply:
> (For example, zulip-mobile has a realm setting `community_topic_editing_limit_seconds` in its type definitions for the API.)

(Meaning both a property `realm_community_topic_editing_limit_seconds` in the /api/register-queue response, and a property `community_topic_editing_limit_seconds` for the data object in a `realm/update_dict` event.)